opmd_viewer/openpmd_timeseries/interactive/__init__.py

```diff
@@ -0,0 +1,22 @@
+# This file imports the class `InteractiveViewer`, which is used
+# as a parent class for the OpenPMDTimeSeries.
+# This parent class defines the method `slider` ; if the dependencies are
+# unavailable, an error message will be returned.
+
+try:
+    # Check wether the interactive interface can be loaded
+    from .interactive import InteractiveViewer
+except ImportError:
+    # Otherwise, create a dummy class which returns an error message
+    # when the method `slider` is called.
+    class InteractiveViewer(object):
```

Comment:
that an ok-ish work-around but one could keep the implementation of the "fallback" `InteractiveViewer` class in `interactive.py` and implement the `try-except` case there. this avoids taking the signature of the class out to the places where it is used